homeassistant/components/nuki/binary_sensor.py

```diff
@@ -70,3 +76,35 @@ def door_sensor_state_name(self):
     def is_on(self):
         """Return true if the door is open."""
         return self.door_sensor_state == STATE_DOORSENSOR_OPENED
+
+
+class NukiRingactionEntity(NukiEntity[NukiDevice], BinarySensorEntity):
+    """Representation of a Nuki Opener Ringaction."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+    _attr_icon = "mdi:bell-ring"
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return f"{self._nuki_device.nuki_id}_ringaction"
+
+    @property
+    def extra_state_attributes(self):
+        """Return the device specific state attributes."""
+        data = {
+            ATTR_NUKI_ID: self._nuki_device.nuki_id,
+            ATTR_RING_TIMESTAMP: self._nuki_device.ring_action_timestamp,
```

Comment:
attributes (especially those which can change frequently) are discouraged to use. That should really be a separate Event entity (separate PR).

Reply:
Alright, attribute is removed now.